content/applications/hr/fleet/configuration.rst

```diff
@@ -0,0 +1,234 @@
+  & Edit`.
+- :guilabel:`Vehicle Type`: Select one of two preconfigured vehicle types, either :guilabel:`Car` or
+  :guilabel:`Bike`, from the drop-down menu. The vehicle types are hardcoded in Odoo and are
+  integrated with the *Payroll* application. Adding additional vehicle types is not possible as it
+  will affect payroll.
+- :guilabel:`Category`: Select the category the vehicle is categorized under from the drop-down
+  menu. To create a new category, type in the category and then click :guilabel:`Create
+  (new category)`.
+
+.. note::
+   When the manufacturer is selected, the logo for the manufacturer will automatically load in the
+   image box in the top right corner.
+
+Information tab
+~~~~~~~~~~~~~~~
+
+Model
+*****
+
+- :guilabel:`Seats Number`: Enter how many passengers the vehicle can accommodate.
+- :guilabel:`Doors Number`: Enter the number of doors the vehicle has.
+- :guilabel:`Color`: Enter the color of the vehicle.
+- :guilabel:`Model Year`: Enter the year the vehicle was manufactured.
+- :guilabel:`Trailer Hitch`: Check this box if the vehicle has a trailer hitch.
+
+Salary
+******
+
+- :guilabel:`Can be requested`: Check this box if employees can request this model vehicle.
+- :guilabel:`Catalog Value (VAT Incl.)`: Enter the :abbr:`MSRP (Manufacturer's Suggested Retail
+  Price)` for the vehicle at the time of purchase or lease.
+- :guilabel:`C02 fee`: This value is specific to Belgian based companies only. If logged into a
+  Belgian company, this value is automatically calculated based on Belgian laws and regulations, and
+  cannot be modified. The value is based on the value entered in the :guilabel:`CO2 Emissions` field
+  in the :guilabel:`Engine` section of the vehicle form. Modifying the :guilabel:`CO2 Emissions`
+  field will adjust the value in the :guilabel:`CO2 fee` field.
+- :guilabel:`Cost (Depreciated)`: Enter the monthly cost for the vehicle, which appears in the
+  salary configurator. This value impacts the gross and net salary of the employee who is assigned
+  to the vehicle. This figure is depreciated over time.
+- :guilabel:`Total Cost (Depreciated))`: This value is the :guilabel:`Cost (Depreciated)` and the
+  :guilabel:`C02 fee` fields combined, and also is depreciated over time.
+
+.. note::
+   The :guilabel:`Cost (Depreciated)` does not depreciate automatically on the vehicle model, it
+   only depreciates based on the contract linked to a specific vehicle and not on the general model.
+
+Engine
+******
+
+- :guilabel:`Fuel Type`: Select the type of fuel the vehicle uses form the drop-down menu. The
+  options are :guilabel:`Diesel`, :guilabel:`Gasoline`, :guilabel:`Hybrid Diesel`, :guilabel:`Hybrid
+  Gasoline`, :guilabel:`Plug-in Hybrid Diesel`, :guilabel:`Plug-in Hybrid Gasoline`,
+  :guilabel:`CNG`, :guilabel:`LPG`, :guilabel:`Hydrogen`, or :guilabel:`Electric`.
+- :guilabel:`CO2 Emissions`: Enter the carbon dioxide emissions the vehicle produces in grams per
```

Comment:
again over life per year? average?

Reply:
it's i grams per kilometer- so it's an average but it's per kilometer (distace) not yearly or monthly (time based). I added it to the second entry to make it more clear.